### DISCUSSION TITLE
Does Noisemodeling not support medium size vehicles as input data?

### DISCUSSION THREAD

**Discussion by mshmoon** [Universite-Gustave-Eiffel/NoiseModelling#626] [General]
![image](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/assets/38131350/01e3d349-2301-4fa6-9ea7-4e3b4caa8d05)
I found there is no MV in the picture

**Comment by nicolas-f** [comment]
Hi,

OpenStreetMap importer does not generate this column.

You can see how it choose the traffic data here:
https://github.com/Universite-Gustave-Eiffel/NoiseModelling/blob/v4.0.5/wps_scripts/src/main/groovy/org/noise_planet/noisemodelling/wps/Import_and_Export/Import_OSM.groovy#L1281

**Reply by mshmoon** [reply]
But can I use noisemadelling to export MV data?

**Reply by pierromond** [reply]
NoiseModelling just computes the noisemap. The input traffic data should be imported by yourself depending of your country and characteristics of your study case.
